config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -124,11 +124,14 @@ find "$base" -name '*.yaml' -exec grep -q 'generated by gen-jobs.sh' '{}' \; -de
 # Resource usage of a job depends on whether it needs to build Kubernetes or not.
 resources_for_kubernetes () {
     local kubernetes="$1"
+    local job_type="$2"
```

Comment:
Instead of `job_type`, add a `local indention="$2"`.

Then do something like this:
```
case ...
   cat <<EOF
resources:
   ...
EOF
   ;;
esac | while read -r line; do echo "${indention}${line}"; done 
```

That avoids the repetition and makes the function more versatile.

Then where it is called, use `$(resources_for_kubernetes "$kubernetes" "    ")` with the right amount of spaces.